src/Microsoft.DotNet.XHarness.CLI/Commands/WebServer.cs

```diff
@@ -65,6 +66,10 @@ internal static async Task<ServerURLs> Start(IWebServerArguments arguments, stri
                     }
                 });
             })
+            .UseConfiguration(
+                new ConfigurationBuilder()
+                .AddEnvironmentVariables(prefix: "BROWSER_")

```

Comment:
Why we are doing it on server side?

Reply:
That was the only idea I came up with to pack setting the LANGUAGE var for the testing process only. I don't see any way of setting it to the BrowserDriver because adding "--lang" argument on Linux it does not have any effect. In my understanding it's the equivalent of setting the env variable for the test process, isn't it?

Reply:
If the browser process needs this environment variable then this needs to be set when launching chrome.

Reply:
Maybe I will paste my test screenshots here from Linux.
Not good:
![image](https://user-images.githubusercontent.com/32700855/217054566-b69e62e9-6a77-41db-b053-1470c04b30f4.png)
Good:
![image](https://user-images.githubusercontent.com/32700855/217055953-654d3a86-3651-4315-af47-1ff96347bd9e.png)

Reply:
Set the environment variable on the process launch then.

Reply:
Credits @radical:
![image](https://user-images.githubusercontent.com/32700855/217487854-3af8be76-2dff-4fdb-b5da-123198a44760.png)